pensjon-brevbaker/src/main/kotlin/no/nav/pensjon/etterlatte/maler/vedlegg/barnepensjon/BeregningAvBarnepensjon.kt

```diff
@@ -258,7 +258,7 @@ private fun OutlineOnlyScope<LanguageSupport.Triple<Bokmal, Nynorsk, English>, B
         }
         paragraph {
             textExpr(
-                Bokmal to "For å få full pensjon må avdødes trygdetid være beregnet til minst 40 år. Trygdetid over 40 år blir ikke tatt med i beregningen. Avdødes samlede trygdetid er beregnet til ".expr() + aarTrygdetid.format() + " år.",
+                Bokmal to "For å få full pensjon må avdødes trygdetid være beregnet til minst 40 år. Trygdetid over 40 år blir ikke tatt med i beregningen. Pensjonen din er beregnet etter trygdetiden til ".expr() + bruktAvdoed  + ". Avdødes samlede trygdetid er beregnet til ".expr() + aarTrygdetid.format() + " år",
```

Comment:
Skal dette endres for alle barnepensjonsbrev som har med beregning? Hva med engelsk og nynorsk isåfall?

Reply:
Skal vel sikkert det egentlig men kan legget til en sjekk på foreldreløs i første omgang